bin/node/runtime/src/lib.rs

```diff
@@ -895,6 +895,22 @@ impl pallet_vesting::Trait for Runtime {
 	type WeightInfo = weights::pallet_vesting::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const AssetDepositBase: Balance = 100 * DOLLARS;
+	pub const AssetDepositPerZombie: Balance = 1 * DOLLARS;
+}
+
+impl pallet_assets::Trait for Runtime {
+	type Event = Event;
+	type Balance = u64;
+	type AssetId = u32;
```

Comment:
nitpick: this should probably be a publicly defined type

Reply:
One extra line for near zero gain. Sounds less like a nit-pick and more like a bikeshed.

Reply:
:shrug: 
These suggestions came out of trying to deploy the pallet and are more quality of life stuff than anything else.